etc/crds/subnets.crd.yaml

```diff
@@ -58,6 +58,11 @@ spec:
       priority: 0
       JSONPath: .spec.vpc
       description: The name of the VPC
+    - name: RemoteDeployed
```

Comment:
This is a very unintuitive name. subnet is a property of a VPC, which in-turn is a property of the cluster. IIRC, in this use-case we are creating a 'shared VPC' across two or more clusters by specifying VNI and designating one or more subnets as common to the shared VPC. Can we call this 'sharedSubnet'?

Reply:
If we deploy 2 subnets in cluster1 and cluster2, are their sharedSubnet both true? How can we know the requests of which subnets are redirected to gateways?

The attribute remoteDeployed is discussed in the previous meeting to show that the subnet is deployed in a remote cluster.

Reply:
A subnet is a range of IP addresses that is a subset of VPC CIDR. So what exactly does it mean to "deploy a range of IP addresses" in a remote cluster?

Reply:
Yes. The subnet applied in cluster1 is actually deployed in the remote cluster2. However, the requests in the cluster1 have to know if the target ips belongs to a remote cluster or not. So that the subnet is not actually in the cluster1 but in the remote cluster2, which is discussed in the previous meeting.